src/migrations/20241002000000-clean-training-report-data.js

```diff
@@ -0,0 +1,41 @@
+const { prepMigration } = require('../lib/migration');
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface) {
+    await queryInterface.sequelize.transaction(async (transaction) => {
+      const sessionSig = __filename;
+      await prepMigration(queryInterface, transaction, sessionSig);
+      return queryInterface.sequelize.query(`
+        UPDATE "SessionReportPilots" SET data = data - 'event';
```

Comment:
do this will remove a property? ie event from session? Are we sure its not being used?

Reply:
Or that it will be put back on next save? Does it makes sense to backup the existing data? Also is it hurting anything being there?

Reply:
its not consistent, and looks to not be populated anymore., So this just cleans up unneeded data.